chia/simulator/block_tools.py

```diff
@@ -849,26 +847,27 @@ def get_consecutive_blocks(
                     sub_slot_iters,
                     True,
                 )
-            if pending_ses:
-                sub_epoch_summary: Optional[SubEpochSummary] = None
-            else:
+            # generate sub_epoch_summary, and if the last block was the last block of the sub-epoch
+            # include the hash in the next sub-slot
+            sub_epoch_summary: Optional[SubEpochSummary] = None
+            if not pending_ses:  # if we just created a sub-epoch summary, we can at least skip another sub-slot
```

Comment:
this is just a pure refactor, right? it doesn't affect any behavior, afaict

Reply:
yeah that part is